packages/preview/oicana/0.1.0/README.md

```diff
@@ -0,0 +1,54 @@
+# Oicana
+*Dynamic PDF Generation based on Typst*
+
+https://oicana.com
+
+Oicana offers seamless PDF templating across multiple platforms. Define your templates in Typst, specify dynamic inputs, and generate high quality PDFs from any environment - whether it's a web browser, server application, or desktop software.
+
+> **Oicana is in Alpha! It is rough around the edges and has a limited number of integrations.**
+
+## What Oicana offers
+
+- *Multi-platform* - The same templates work with all Oicana integrations.
+- *Powerful Layouting* - Templates can use all of Typst's functionality including its extensive package ecosystem.
+- *Performant* - Create a PDF in single digit milliseconds.
+- *Version Control Ready* - Templates are mostly text files and can live next to your source code.
+- *Escape Vendor Lock-in* - Reuse templates with other Typst based solutions. The Typst compiler is open source!
+
+## Example usage
```

Comment:
I think it would help a lot if you'd add an example of a manifest. Just a minimal manifest with the required keys would clear up how things are supposed to work.